src/components/Ditto.tsx

```diff
@@ -11,27 +11,42 @@ import {
   isProject,
 } from "../lib/utils";
 
+export interface Variables {
+  [variableId: string]: string
+}
```

Comment:
Looks like this interface is duplicated inside of `lib/utils`; I think we should get rid of the duplicate definition and only keep one of them

Reply:
👍